honeycombextension/extension.go

```diff
@@ -43,12 +43,10 @@ const (
 	logs    = signal("logs")
 )
 
-func newBytesReceivedMap() map[signal][]datapoint {
-	return map[signal][]datapoint{
-		traces:  make([]datapoint, 0),
-		metrics: make([]datapoint, 0),
-		logs:    make([]datapoint, 0),
-	}
+type bytesReceivedMap map[signal][]datapoint
+
+func newBytesReceivedMap() bytesReceivedMap {
+	return make(map[signal][]datapoint, 0)
```

Comment:
I can't remember, does this initialize the slice as well? I want to make sure 

```go
h.bytesReceivedData[traces] = append(h.bytesReceivedData[traces], datapoint{timestamp: time.Now(), value: int64(size)})
```

still works as it expects `h.bytesReceivedData[traces]` to be an initialized slice.

Reply:
`append` will initialize the the slice if it's nil